cloudformation.yaml

```diff
@@ -156,3 +156,8 @@ Outputs:
   LambdaFunction:
     Description: 'Lambda function name created by this stack.'
     Value: !Ref Function
+  LambdaFunctionARN:
+    Description: 'Lambda function ARN created by this stack.'
+    Value: !GetAtt Function.Arn
+    Export:
+      Name: aws-to-slack-arn
```

Comment:
By using a fixed-name export, this stack can only be created once within a region!  We already have users (including myself) that create multiple stacks to route messages to different channels.

Please change to:
```
    Export:
        Name: !Sub '${AWS::StackName}-FunctionArn'
```